### ISSUE TITLE
Question about llama

### ISSUE THREAD

**Issue by Wgkai** [rese1f/MovieChat#74]
https://github.com/rese1f/MovieChat/blob/a2d0da02f6cf16383ae1f9891ecf62e6f8f798a5/MovieChat/models/moviechat.py#L156C9-L176C1

**Comment by Espere-1119-Song** [comment]
Sorry for the confusing code. In fact, the weight of projection layer is loaded after the model initialized.
<img width="726" alt="image" src="https://github.com/user-attachments/assets/65000fa7-4734-4385-b1aa-abfb0d6ef698">

**Comment by xuzq23** [comment]
I have a similar confusion, 
the model path setting in the eval_configs/MovieChat.yaml
```
  llama_proj_model: 'ckpt/minigpt4/pretrained_minigpt4.pth'
  ckpt: "ckpt/finetune-vicuna7b-v2.pth"
```
according to your explanation above, would the `llama_proj_model ` and `ckpt`(line [690)](https://github.com/rese1f/MovieChat/blob/a2d0da02f6cf16383ae1f9891ecf62e6f8f798a5/MovieChat/models/moviechat.py#L690) take the same effect?

**Comment by Espere-1119-Song** [comment]
Yes, they take the same effect.`llama_proj_model` seems not used in the code, and we will revise it.

**Comment by xuzq23** [comment]
> Yes, they take the same effect.`llama_proj_model` seems not used in the code, and we will revise it.

Thank you sincerely.
